dotnet/src/IntegrationTests/prompts/GenerateStory.yaml

```diff
@@ -0,0 +1,16 @@
+﻿name: GenerateStory
+template: |
+  Tell a story about {{$topic}} that is {{$length}} sentences long.
+template_format: semantic-kernel
+description: A function that generates a story about a topic.
+input_variables:
+  - name: topic
+    description: The topic of the story.
+    is_required: true
+  - name: length
+    description: The number of sentences in the story.
+    is_required: true
+output_variable:
+  description: The generated story.
+execution_settings:
+    - temperature: 0.6
```

Comment:
It seems there are extra spaces in line #16. Please double-check that the temperature is taken into account and the default is not applied.